src/type-inference.md

```diff
@@ -1,18 +1,18 @@
 # Type inference
 
-The type inference is based on standard HM-type inference, but
-extended in various way to accommodate subtyping, region inference,
+The type inference is based on the standard Hindley–Milner (HM) system,
+but extended in various way to accommodate subtyping, region inference,
 and higher-ranked types.
 
 ## A note on terminology
 
 We use the notation `?T` to refer to inference variables, also called
 existential variables.
 
-We use the term "region" and "lifetime" interchangeably. Both refer to
+We use the terms "region" and "lifetime" interchangeably. Both refer to
 the `'a` in `&'a T`.
 
-The term "bound region" refers to regions bound in a function
+The term "bound region" refers to a region bound in a function
```

Comment:
Perhaps "a region that is bound" is clearer?

Reply:
I agree.